app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/DutiesSelectedAnswer.scala

```diff
@@ -16,60 +16,35 @@
 
 package uk.gov.hmrc.cdsreimbursementclaimfrontend.models
 
-import cats.kernel.Eq
-import julienrf.json.derived
-import play.api.libs.json.OFormat
-import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.form.{DutiesSelected, Duty}
-
-sealed trait DutiesSelectedAnswer extends Product with Serializable
+import cats.data.NonEmptyList
+import play.api.libs.json.Format
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.form.Duty
 
 object DutiesSelectedAnswer {
 
-  final case class IncompleteDutiesSelectedAnswer(
-    maybeDutiesSelected: Option[DutiesSelected]
-  ) extends DutiesSelectedAnswer
-
-  object IncompleteDutiesSelectedAnswer {
-    val empty: IncompleteDutiesSelectedAnswer           = IncompleteDutiesSelectedAnswer(None)
-    implicit val eq: Eq[IncompleteDutiesSelectedAnswer] = Eq.fromUniversalEquals[IncompleteDutiesSelectedAnswer]
+  type DutiesSelectedAnswer = NonEmptyList[Duty]
 
-    implicit val format: OFormat[IncompleteDutiesSelectedAnswer] =
-      derived.oformat[IncompleteDutiesSelectedAnswer]()
-  }
+  implicit val nelFormat: Format[NonEmptyList[Duty]] = Format(NonEmptyListOps.reads, NonEmptyListOps.writes)
 
-  final case class CompleteDutiesSelectedAnswer(
-    dutiesSelected: DutiesSelected
-  ) extends DutiesSelectedAnswer
+}
 
-  object CompleteDutiesSelectedAnswer {
-    implicit val format: OFormat[CompleteDutiesSelectedAnswer] =
-      derived.oformat[CompleteDutiesSelectedAnswer]()
-  }
+import play.api.libs.json._
+object NonEmptyListOps {
```

Comment:
This 
```
implicit val nelFormat: Format[NonEmptyList[Duty]] = Format(NonEmptyListOps.reads, NonEmptyListOps.writes)
```
can be generalised by having
```
  implicit def nelFormat[A : Format]: Format[NonEmptyList[A]] =
    Format(nelReads, nelWrites)
```
There is no need to define it every time for every collection.

Also I would recommend moving it to the package file, perhaps `formats`/`models`?
Something like this
```
package object models {

  //--formats--

  def nelReads[A : Reads]: Reads[NonEmptyList[A]] =
    Reads.of[List[A]].collect(JsonValidationError("Expected a non empty list but got an empty list")) { case x :: xs =>
      NonEmptyList(x, xs)
    }

  def nelWrites[A : Writes]: Writes[NonEmptyList[A]] =
    Writes.of[List[A]].contramap(_.toList)

  implicit def nelFormat[A : Format]: Format[NonEmptyList[A]] =
    Format(nelReads, nelWrites)
}
```
The benefits are the similar as above, no need for an explicit import in your code. So currently developers must do
```
import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.NonEmptyListOps._
```
To make serialisation happen. 
Perhaps making it to happen automatically would help everyone.

Reply:
Also naming `NonEmptyListOps` containing `Ops` in the end normally is associated with extension classes. For example, here is a `StringOps` from vanilla Scala to extend Java's String class - https://www.scala-lang.org/api/2.12.2/scala/collection/immutable/StringOps.html
This pattern: Pimp-my-library - https://kubuszok.com/2018/implicits-type-classes-and-extension-methods-part-3/ got so popular so that even found it continuation in Scala 3 as extensions methods:
```
case class Circle(x: Double, y: Double, radius: Double)

extension (c: Circle)
  def circumference: Double = c.radius * math.Pi * 2
```
http://dotty.epfl.ch/docs/reference/contextual/extension-methods.html

So maybe it is not great naming :)

Reply:
You are right. I just grabbed this piece of code from github in a hurry, fixed it up in a few minutes and then forgot about it.
This often happens when I feel under time pressure to deliver at the end of the sprint